src/submission/forms.py

```diff
@@ -432,3 +438,25 @@ def __init__(self, *args, **kwargs):
     class Meta:
         model = models.Article
         fields = ('projected_issue',)
+
+
+def utility_clean_orcid(form, field_name='orcid'):
+    """
+    Utility function that cleans an ORCID ID.
+    """
+    orcid = form.cleaned_data[field_name]
```

Comment:
If I'm reading the below correctly, this will still return the uncleaned value if a malformed orcid is passed. Is that desired?

Reply:
Yes I believe so as the malformed value is shown to the user with the error.